qa/rpc-tests/fundrawtransaction.py

```diff
@@ -572,7 +575,14 @@ def run_test(self):
         # test fundrawtransaction with OP_RETURN and no vin #
         #####################################################
 
-        rawtx   = "0100000000010000000000000000066a047465737400000000"
+        # NAVCoin: version number is 02000000 (4-bytes)
+        # NAVCoin: time field is 01000000 (4-bytes)
+        # NAVCoin: vin number is 00
+        # NAVCoin: vout number is 01
+        # NAVCoin: vout value is 0000000000000000 (8-bytes)
+        # NAVCoin: vout_length is 06 (1-byte)
+        # NAVCoin: OP_RETURN script is 6a0474657374
```

Comment:
transactions with version >= 2 include an additional strdzeel parameter

Reply:
What does this mean for this test? we need to modify these bytes?